disperser/common/blobstore/blob_metadata_store.go

```diff
@@ -117,10 +124,14 @@ func (s *BlobMetadataStore) GetBlobMetadataByStatus(ctx context.Context, status
 // Because this function scans the entire index, it should only be used for status with a limited number of items.
 // It should only be used to filter "Processing" status. To support other status, a streaming version should be implemented.
 func (s *BlobMetadataStore) GetBlobMetadataByStatusCount(ctx context.Context, status disperser.BlobStatus) (int32, error) {
```

Comment:
This function name is a bit odd. It should probably be `GetBlobCountByStatus` or `GetBlobMetadatCountByStatus`.

Reply:
Yeah updated to `GetBlobMetadatCountByStatus`